### DISCUSSION TITLE
Row input sidebar freezes or is unresponsive loading

### DISCUSSION THREAD

**Discussion by wmacklin** [AntlerVC/firetable#908] [Support / Q&A] [answered]
I believe this is a platform issue, The Row input sidebar freezes when loading data from the row. I removed all data and started fresh. I even used a different browser, Not sure what the problem is, Does anyone have this problem and how can I fix it?
![image](https://user-images.githubusercontent.com/11440537/198892199-e7cd3d0c-6296-490d-b158-67815a2ea4db.png)

**Comment by notsidney** [comment]
Hi @wmacklin, do you happen to have a ref field type in your row, or store a ref in the document?

Your screenshot shows the side drawer is loading in the edit components. This happens the first time you open it, so that opening the table is fast. If you have a slow internet connection, loading may take a while. How long does it appear frozen for?

**Comment by BeeBombshell** [comment]
Hey @wmacklin, I can make out why this is happening for you. I believe there are too many fields on the sidebar which are causing this issue. Try hiding a few of the columns that aren't required currently. This will reduce memory consumption and hopefully the sidebar would work perfectly.

**Comment by harinij** [comment, marked as answer]
This issue should be fixed in our newer releases. Let us know if you are still facing this issuing. Closing this out for now.
